documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -71,7 +71,40 @@ A purpose declares what the application intends to do with a set of personal inf
 
 ### Applying purpose concept in the authorization request
 
->[TO BE EDITED/COMPLETED] - [OAuth 2.0 Authorization Framework](https://www.rfc-editor.org/rfc/rfc6749)
+In order to declare a purpose when accessing CAMARA APIs, it is strongly recommended that `scope` param in the /authorize request will be valued with:
```

Comment:
> in the /authorize request

Is this proposal only intended to apply to the front-end flow? CIBA flows and client credentials require scopes as well! Surely this proposal applies to all authorisation schemes.

Reply:
Yes, it applies to all authorisation schemes. Let me rephrase that a bit, as I tried to keep the original github comment to avoid any misunderstanding. But you've raised a fair point. May be just removing that /authorize part is enough... you can also see that, for example, the auth code flow and the CIBA flow are already updated with this agreement.